tests/fixtures/scenarios/vendor-pulls-ssh/atmos.yaml

```
@@ -0,0 +1,40 @@
+logs:
+  file: "/dev/stderr"
+  level: Info
+
+# Custom CLI commands
+
+# No arguments or flags are required
+commands:
+- name: "test"
+  description: "Run all tests"
+  steps:
+  - atmos vendor pull --everything
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)